src/server/supplements/recorderSupplement.ts

```diff
@@ -456,12 +456,17 @@ export class RecorderSupplement {
     this._recorderApp?.setSources([...this._recorderSources, ...this._userSources.values()]);
   }
 
-  onBeforeInputAction(sdkObject: SdkObject, metadata: CallMetadata) {
+  async onBeforeInputAction(sdkObject: SdkObject, metadata: CallMetadata) {
     if (this._mode === 'recording')
       return;
     this._captureSnapshot(sdkObject, metadata, 'action');
   }
 
+  async onCallLog(logName: string, message: string, sdkObject: SdkObject, metadata: CallMetadata): Promise<void> {
+    debugLogger.log(logName as any, message);
```

Comment:
It seems like we've lost logging when RecorderSupplement is not created (which I assume is most of the time)?